app/lib/actions/katello/content_view_version/destroy.rb

```diff
@@ -4,15 +4,18 @@ module ContentViewVersion
       class Destroy < Actions::Base
         def plan(version, options = {})
           version.check_ready_to_destroy! unless options[:skip_environment_check]
+          destroy_env_content = !options.fetch(:skip_destroy_env_content, false)
+          repos = destroy_env_content ? version.repositories : version.archived_repositories
```

Comment:
The problem was that in ContentView::Remove, both CVE delete and CVV delete were trying to delete the same sets of repos. CVV delete was changed to delete env repos since it's used by org destroy. However, ContentView::Remove removes all CVEs before removing the CVV so we were getting a 404 from pulp. This adds an option CV::Remove can use to skip deleting a CVV's env content.